python/cudf/cudf/utils/ioutils.py

```diff
@@ -1642,6 +1585,7 @@ def _set_context(obj, stack):
     return stack.enter_context(obj)
 
 
+# TODO: May be able to delete this now.
```

Comment:
Pretty sure you can - My vote is that you go ahead and remove this :)

Reply:
(Also happy to add the `_open_remote_files` removal to https://github.com/rapidsai/cudf/pull/16577 after this Pr goes in)

Reply:
Good eye. I forgot about this. I'll remove it here.